client/platform/web-girder/views/ViewerAlert.vue

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+import { defineComponent, ref, toRef } from '@vue/composition-api';
+
+export default defineComponent({
+  name: 'ViewerAlert',
+  setup(_, { root }) {
+    const dialog = ref(false);
+
+
+    // this will change once REST updates are merged
+    const brandData = toRef(root.$store.state.Brand, 'brandData');
+
+    return {
+      dialog,
+      brandData,
+    };
+  },
+});
+</script>
+
+<template>
+  <!-- This component is for alert messages on the Viewer Page only.
+    See NavigationBar for sitewide alert banner -->
+  <v-dialog
+    v-model="dialog"
+    max-width="600px"
+  >
+    <template v-slot:activator="{ on, attrs }">
+      <v-icon
+        v-if="brandData.alertMessage"
+        dark
+        large
+        color="red"
+        v-bind="attrs"
+        v-on="on"
+      >
+        mdi-alert-circle
+      </v-icon>
+    </template>
+    <v-card
+      color="blue"
+      class="pa-4"
+    >
+      <v-icon
+        class="float-right"
+        @click="dialog = false"
+      >
+        mdi-close
+      </v-icon>
+      {{ brandData.alertMessage }}
+    </v-card>
+  </v-dialog>
+</template>
+
+<style lang='scss'>
+.theme--dark.v-dialog--active {
+  background: green;
+}
+
+</style>
```

Comment:
Generally best to avoid hacking into private CSS classes in vuetify as they make version upgrades difficult and require you to use unscoped (global) css.  These classes are not part of the Vuetify public API, so they may change without warning.

This block will also effect every `v-card` in the application because it's unscoped.